記事を投稿

ZabtonChino opened the pull request:
postにdefine-contという名前で，mdファイルを追加しました．記法や内容の修正を行うので，確認次第指示をいただければ幸いです．

ZabtonChino committed:
Merge pull request #3 from Y-Naka-implicit/new-post

記事を投稿

ynu-math commented on _posts/2021-04-07-define-cont.md:
```diff
@@ -0,0 +1,271 @@
+
+## $\varepsilon\delta$論法による連続性の定義
+
+ここでは，距離空間の連続写像を考えるために，高校での連続の定義の問題点を挙げ，$\varepsilon\delta$論法による連続の定義を行う．
```
`$\varepsilon - \delta$論法`のようにハイフンを入れるほうが一般的だろうと思います．間違いではありません．

ynu-math commented on _posts/2021-04-07-define-cont.md:
```diff
@@ -0,0 +1,271 @@
+
+定義(?)．
+:   $a$を$f(x)$の定義域に属する値とするとき，関数$f(x)$が$x=a$で連続であるとは，次の2つのことが満たされていることである．
+1. 極限値$\lim_{x\to a}f(x)$が存在する．
```
`$\displaystyle \lim_{x\to a}f(x)$`としたほうが見やすいです．

ynu-math commented on _posts/2021-04-07-define-cont.md:
```diff
@@ -0,0 +1,271 @@
+:   $a$を$f(x)$の定義域に属する値とするとき，関数$f(x)$が$x=a$で連続であるとは，次の2つのことが満たされていることである．
+1. 極限値$\lim_{x\to a}f(x)$が存在する．
+2. $\lim_{x\to a}f(x)=f(a)$が成立する．
+大島利雄ほか13名著 (2018).極限,『改訂版　数学III』,数研出版,pp.141-142.より引用．
```
脚注は
```tex
定義(?)．
:   $a$を$f(x)$の定義域に属する値とするとき，関数$f(x)$が$x=a$で連続であるとは，次の2つのことが満たされていることである．[^oshima]
    1. 極限値$\lim_{x\to a}f(x)$が存在する．
    2. $\lim_{x\to a}f(x)=f(a)$が成立する．

[^oshima]: 大島利雄ほか13名著 (2018).極限,『改訂版　数学III』,数研出版,pp.141-142.より引用
```
のようにするとかけます．

Commit:
pull requestにて指摘された箇所を修正

ZabtonChino committed:
Merge pull request #4 from Y-Naka-implicit/new-post

pull requestにて指摘された箇所を修正

ZabtonChino commented:
指摘箇所を修正しました．確認をお願いします．

ynu-math commented on _posts/2021-04-07-define-cont.md:
```diff
@@ -0,0 +1,266 @@
+ノルムは，ベクトルそのものの大きさを測る写像である．ユークリッド空間以外の空間でも連続写像を考えるためには，2点間の距離を直接測れる方がよい．ここではユークリッド空間の距離を考察し，一般の距離空間を定義する．
+
+
+$(1,1)\in \mathbb{R}^2$と$(2,3)\in \mathbb{R}^2$の距離は，$\sqrt{(2-1)^2+(3-1)^2}=\sqrt{1+4}=\sqrt{5}$と計算される．これを$d((1,1),(2,3))$と書けば，距離は，$d:\mathbb{R}^2\times \mathbb{R}^2\to \mathbb{R}$なる写像を考えているということである．ここからの一般化として，ユークリッド空間の間の距離を定義する．
```
`ここからの一般化として，ユークリッド空間の間の距離を定義する．`
とありますが，「ユークリッド空間における距離」の方が適切かなと思います．

shibaleo committed:
Merge remote-tracking branch 'upstream/gh-pages' into gh-pages

shibaleo committed:
定理環境のようなものを追加

shibaleo committed:
environmentを設定

- {% include definition.html content=" 内容"%}のように使って枠囲み環境を呼び出す

shibaleo committed:
見た目のみ修正

Commit:
comflictを解消

Commit:
Merge branch 'gh-pages' of github.com:Y-Naka-implicit/ynu-math.github.io

ZabtonChino committed:
Merge branch 'gh-pages' into gh-pages

ZabtonChino committed:
Merge pull request #5 from shiba-dog-leo/gh-pages

記事の内容に合わせてjekyllをアップデートしました．

comflictを解消しました．

shibaleo committed:
記事を改変

ZabtonChino committed:
Merge pull request #6 from shiba-dog-leo/gh-pages

記事を改変

shibaleo committed:
記事を推敲

- 日本語の文章として気になるところを修正しました．

shibaleo committed:
誤字を修正

shibaleo committed:
記事を修正

ZabtonChino committed:
Update 2021-04-07-define-cont.md

330行目を修正．
注意コメントを削除．

ZabtonChino committed:
Update 2021-04-07-define-cont.md

325行目からの文章を修正．

shibaleo committed:
Merge branch 'master' into gh-pages

shibaleo committed:
Merge branch 'gh-pages' of github.com:shiba-dog-leo/ynu-math.github.io into gh-pages

shibaleo committed:
記事の内容を修正

- 途中途中で話題がずれていると思われるものは軌道を修正しました．
たとえばユークリッドノルムなどの概念は，距離の話と混同する恐れがあるので記述を削除しました．
- 「連続性を抽象化する」などの記述があったため，「距離を抽象化する」に修正しました．
- 他にも「差」→「距離」などと言い換えた部分があるので，用語の使い分けに注意して差分をご覧ください

ZabtonChino committed:
Merge pull request #7 from shiba-dog-leo/gh-pages

記事を推敲

要改善だが修正のため一旦mergeします．

Commit:
「距離空間における点列の収束について」の節を追加．それに伴って，参照の文面を修正．
